src/org/parosproxy/paros/extension/history/PopupMenuExportResponse.java

```diff
@@ -102,7 +85,16 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
         	    BufferedOutputStream bos = null;
                 try {
                     bos = new BufferedOutputStream(new FileOutputStream(file, isAppend));
-           	        exportHistory(msg, bos);
+                    for (HistoryReference href : hrefs) {
+                        HttpMessage msg = null;
+                        try {
+                            msg = href.getHttpMessage();
+                        } catch (Exception e1) {

```

Comment:
Sorry, it's probably better to not catch the exception, that way the user is informed (UI) that an error occurred.

Reply:
So totally remove the catch or put the dialog back in?

Reply:
Remove the catch (the exception is being caught in a following catch block).